Rubberduck.Inspections/Concrete/EmptyConditionBlockInspection.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Inspections.Abstract;
+using Rubberduck.Parsing.Inspections.Abstract;
+using Rubberduck.Parsing.Inspections.Resources;
+using Antlr4.Runtime;
+using Antlr4.Runtime.Misc;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.Inspections.Results;
+
+namespace Rubberduck.Inspections.Concrete
+{
+    [Flags]
+    public enum ConditionBlockToInspect
```

Comment:
This should be `ConditionBlockTypeToInspect`.  The current name suggests an actual code block.